cmd/utils.go

```diff
@@ -75,11 +75,22 @@ func getDepInfo() *DependencyOverview {
 
 		// anything where the LHS is not mainModule
 		// is a transitive dependency
```

Comment:
accumulating directDeps here would be easier to do, I think.

There are three possibilities here:

```
switch {
case lhs and rhs are in mainModules:
  ignore
case lhs is in mainModules:
  add to directDeps list
case lhs is not in mainModules:
  add to transDeps list
}
```